src/installer/test/Microsoft.NET.HostModel.Tests/Helpers/BundleHelper.cs

```diff
@@ -22,6 +22,11 @@ public static string GetAppPath(TestProjectFixture fixture)
             return Path.Combine(GetPublishPath(fixture), GetAppName(fixture));
         }
 
+        public static string GetPublishedSingleFilePath(TestProjectFixture fixture)
+        {
+            return GetHostPath(fixture);
```

Comment:
Do we really need a separate function for this if all it is calling is `GetHostPath`?

Reply:
I do think it is useful to have the wrapper call. In the single-file tests, we generate single-file apps in two ways: 
* Through the SDK (dotnet publish /p:PublishSingleFile=true)
* Through invoking HostModel functions directly.

In the former case GetHostPath() will be the pre-bundled app, whereas in the later case, GetHostPath() will just be the host. So, I think it is easier to refer to them by different names.
Helps documenting the intent, and in searching for the use with different semantics.

Reply:
Ahh, okay. I didn't realize the different use cases and just assumed the paths in `BundleHelper` were for the pre-bundled app.